app/theme/client/imports/components/popover.css

```diff
@@ -32,10 +32,12 @@
 	&__content {
 		position: absolute;
 
-		display: flex;
+		display: -webkit-box;
```

Comment:
Why -webkit-box?

Reply:
I didn't notice the `flex-direction` property. Only `display: -webkit-box` seemed to work. I have corrected the error.